orbs/shared/jobs/test.yaml

```diff
@@ -7,6 +7,9 @@ parameters:
     default: _json_key
     description: Username to use when fetching images from a registry
     type: string
+  pre_setup_script:
+    description: Environment variable that contains a shell path to execute before running tests in CircleCI
```

Comment:
Docs tweaks:
```suggestion
    description: Environment variable that contains an executable to run before running tests in CircleCI. This may not include arguments.
```